scripts/vr-test.js

```diff
@@ -7,7 +7,9 @@ const ci = process.argv.some((arg) => arg === '--ci');
 
 const baseCommand =
   'cd ../testing/visual-regression && npx backstop test --docker';
-const command = ci ? `${baseCommand} --config=backstop-ci.json` : baseCommand;
+const command = ci
+  ? `${baseCommand} --config=backstop-config-ci.json`
```

Comment:
Whis should be `backstop-config-ci.js` right?

Reply:
Yes it should!